ArkEcosystem/explorer
# fix: sync navbar price and price blocks in home page

## Summary

https://app.clickup.com/t/1jkmgr8
The issue is because the component poll the price in different moments, one every 8 secs and the other one every minute 

What I'm doing is to completely remove the wire:poll attribute and triggering the price update with a single JS interval so both are always updated at the same time


To test:
- open the page, check the network request and see how every 8 seconds it updates both prices  

## Checklist
